src/makeRobot.js

```diff
@@ -38,7 +38,78 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+
+    get info() {
+      return `name: ${this.name}, \
+chip version: ${this.version}, \
+wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.x}, y=${this.y}`;
+    },
+    get coords() {
+      return {
+        x: this.x,
+        y: this.y,
+      };
+    },
+    goForward(y) {
+      if (y > 0) {
+        this.y = this.y + y;
+      }
+
+      if (y === undefined) {
+        this.y = this.y + 1;
+      }
+
+      return this;
+    },
```

Comment:
Ці функції потрібно спростити 
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.y += y;
      }

      return this;
    },
```